src/users/routes.py

```diff
@@ -17,6 +17,17 @@
 bp = Blueprint("users", __name__, template_folder="templates")
 
 
+def get_current_user() -> User | None:
```

Comment:
This change had to be in the scope of separating user-related functionality (#77). And not in the scope of this ticket.

Since I missed it on the review, I've done this in a separate ticket #87 and merged to develop. So after you rebase on the last develop branch, you won't need this change.